content/diplomacy/gift-empire/plugin.js

```diff
@@ -144,4 +144,4 @@ class Plugin {
   }
 }
 
-plugin.register(new Plugin());
+export default plugin;
```

Comment:
```suggestion
export default Plugin;
```

Reply:
🤦 I was doing this late at night, thanks for the catch.